pkg/diagnose/cni.go

```diff
@@ -206,3 +222,101 @@ func mustHaveSubmariner(clusterInfo *cluster.Info) {
 		panic("cluster.Info.Submariner field cannot be nil")
 	}
 }
+
+func checkOVNVersion(info *cluster.Info, status reporter.Interface) bool {
+	status.Start("Checking OVN version")
+	defer status.End()
+
+	clientSet := info.ClientProducer.ForKubernetes()
+
+	ovnPod, err := mustFindPod(clientSet, ovnKubeDBPodLabel)
+	if err != nil {
+		status.Failure("Failed to get OVNKubeDB Pod %v", err)
+		return false
+	}
+
+	ovnNBVersion, err := getOVNNBVersion(clientSet, info.RestConfig, ovnPod)
+	if err != nil {
+		status.Failure("Failed to get OVN NB version %v", err)
+		return false
+	}
+
+	if version.Compare(ovnNBVersion, minOVNNBVersion, "<") {
+		status.Failure("The OVN NB DB version %v is less than the minimum supported version %v", ovnNBVersion, minOVNNBVersion)
```

Comment:
```suggestion
		status.Failure("The ovn-nb database version %v is less than the minimum supported version %v", ovnNBVersion, minOVNNBVersion)
```

Looking at the man pages, the convention seems to be ovn-nb database.